tsconfig.json

```diff
@@ -6,10 +6,10 @@
     "resolveJsonModule": true,
     "rootDir": ".",
     "useUnknownInCatchVariables": false,
-    "declaration": false,
```

Comment:
This is now an error and it had to be removed. We can set `declarationDir` if we want to output the typings into another directory, and then ignore them.

Reply:
Yes, we should do that. We want to release with the typedoc from the language repo.

Reply:
alright, I pushed them to `_types` at the repo's root. using underscore to suggest they are ignored, although not sure if that convention is valid here :P

I could also put it under `dist/_types`. I believe nothing would read the types unless we add the path somewhere. However, these types are not distributed, so that didn't make much sense to me. I also noticed that there _is_ already a `dist/types` and I didn't want those types to be mixed with these types.